deployments/helm/nvidia-dra-driver-gpu/values.yaml

```diff
@@ -61,6 +61,8 @@ resources:
     enabled: true
 
 controller:
+  labels:
+    app: nvidia-dra-driver-gpu
```

Comment:
For example `ComputeDomainLabelKey` is only applied to managed operands, but not to the operator. 
currently both operators `{controller,kubeletplugin}` are unlabeled

Reply:
now with a flag